Source/WebCore/page/IntersectionObserver.idl

```diff
@@ -23,7 +23,7 @@
  * THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-// https://wicg.github.io/IntersectionObserver/
+// https://www.w3.org/TR/intersection-observer/#intersection-observer-interface
```

Comment:
Please use the link to https://w3c.github.io/IntersectionObserver/ instead

Reply:
Done in next commit `9298eb6`.